solr/FULLSTORY-CHANGES.txt

```diff
@@ -3,4 +3,5 @@
 * fs.ignoreconflicts (Scott Blum)
 * FullStory: special OOMKILLER handling for spaces (Scott Blum)
 * Remove pathological error reporting. Don't serialize the entire clusterstate to log (Scott Blum)
-* Expose DocValuesTermsCollector and TermsQuery (Jaime Yap)
\ No newline at end of file
+* Expose DocValuesTermsCollector and TermsQuery (Jaime Yap)
+* LUCENE-8725: Make TermsQuery.SeekingTermSetTermsEnum a top level class and public
```

Comment:
We should go back through and carefully curate this file at some point.  This would definitely subsume exposing TermsQuery, and I think "Don't serialize the entire clusterstate to log" is actually upstreamed now?

Reply:
> Don't serialize the entire clusterstate to log" is actually upstreamed now?

yes.